perfkitbenchmarker/package_managers.py

```diff
@@ -161,6 +161,8 @@ def InstallPackageGroup(self, package_group):
 
   def Install(self, package_name):
     """Installs a PerfKit package on the VM."""
+    if self.is_static and not self.install_packages:
```

Comment:
Is these a reason to restrict this to only static VMs?  I can see this being useful for even custom images.